bokehjs/src/coffee/renderer/annotation/box_annotation.coffee

```diff
@@ -50,6 +50,7 @@ class BoxAnnotation extends HasParent
 
   defaults: ->
     return _.extend {}, super(), {
+      plot: null
```

Comment:
I didn't know whether it was a goal to sync the client and server class hierarchies... I can add an automated verify of that as well...